### PR TITLE
Update to fixe french labels for System-wide alert

### PR DESCRIPTION
**Pull request by pilasou**
Add translations for the System wide alert feature as well other various missing parameters.

### CODE REVIEW

File: src/assets/i18n/fr.json5 [hunk 1]
```diff
@@ -6074,5 +6179,81 @@
 
   // "idle-modal.extend-session": "Extend session"
   "idle-modal.extend-session": "Prolonger la session"
```
Comment (tdonohue) [inline src/assets/i18n/fr.json5:6181]: I think the issue is on this line... there should be a comma at the end of this line

**Review by tdonohue — CHANGES_REQUESTED** [review]
@pilasou : Thanks for the PR!  Currently though this is failing to build.  It looks like the JSON5 file is accidentally invaild.  I'm seeing this error when I try to build:
```
Error: JSON5: invalid character '\"' at 6186:3
```

I think I found the issue, see my comment below.  Once it's fixed, I can re-test.

File: src/assets/i18n/fr.json5 [hunk 2]
```diff
@@ -2495,6 +2513,71 @@
...
+  "item.edit.identifiers.doi.status.TO_BE_REGISTERED": "En attente d'inscription",
+
+  // "item.edit.identifiers.doi.status.TO_BE_RESERVED": "Queued for reservation",
+  "item.edit.identifiers.doi.status.TO_BE_RESERVED": "En attente de réservation",
+
+  // "item.edit.identifiers.doi.status.IS_REGISTERED": "Registered",
+  "item.edit.identifiers.doi.status.IS_REGISTERED": "Inscrit",
+
+  // "item.edit.identifiers.doi.status.IS_RESERVED": "Reserved",
+  "item.edit.identifiers.doi.status.IS_RESERVED": "Réseré",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:2530]: Should be "Réservé" instead of "Réseré".

File: src/assets/i18n/fr.json5 [hunk 3]
```diff
@@ -2495,6 +2513,71 @@
...
+  "item.edit.tabs.status.buttons.register-doi.label": "Inscrire un nouveau DOI ou un DOI en attente",
+
+  // "item.edit.tabs.status.buttons.register-doi.button": "Register DOI...",
+  "item.edit.tabs.status.buttons.register-doi.button": "Inscrire le DOI...",
+
+  // "item.edit.register-doi.header": "Register a new or pending DOI",
+  "item.edit.register-doi.header": "Inscrire un nouveau DOI ou un DOI en attente",
+
+  // "item.edit.register-doi.description": "Review any pending identifiers and item metadata below and click Confirm to proceed with DOI registration, or Cancel to back out",
+  "item.edit.register-doi.description": "Réviser les identifiants en attente and les métadonnées ci-dessous item metadata below puis cliquer sur Confirmer afin de lancer l'inscription du DOI ou sur Annuler pour interrompre l'inscription",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:2563]: Should be : "Réviser les identifiants en attente and les métadonnées ci-dessous puis cliquer sur Confirmer afin de lancer l'inscription du DOI ou sur Annuler pour interrompre l'inscription"

File: src/assets/i18n/fr.json5 [hunk 4]
```diff
@@ -5991,14 +6086,24 @@
   // "virtual-metadata.delete-relationship.modal-head": "Select the items for which you want to save the virtual metadata as real metadata",
   "virtual-metadata.delete-relationship.modal-head": "Sélectionner les Items pour lesquels vous souhaitez sauvegarder les métadonnées virtuelles en tant que métadonnées réelles",
 
+  
+  //"supervisedWorkspace.search.results.head": "Supervised Items",
+  "supervisedWorkspace.search.results.head": "Items supervisés",
```
Comment (mhvezina) [inline src/assets/i18n/fr.json5:6091]: Suggestion : "Documents supervisés" or "Éléments supervisés".

**Pull request merged**